java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStore.java

```diff
@@ -127,6 +128,65 @@ public void addFiles(List<FileReference> fileReferences) throws StateStoreExcept
         }
     }
 
+    @Override
+    public void splitFileReferences(List<SplitFileReferenceRequest> splitRequests) throws StateStoreException {
+        // Each splitRequest performs 4 transactions in DynamoDB
```

Comment:
That's only true if the file is split into 2. Currently that's what happens, but in future we might split a file into 4 or 8.